athena/llm_core/llm_core/models/providers/openai_model_config.py

```
@@ -0,0 +1,31 @@
+    ENUM: ClassVar[type] = OpenAIModel
+    KW_REMAP: ClassVar[dict[str, str]] = {
+        "max_tokens": "max_completion_tokens",
+    }
+
+    provider: Literal["openai"] = Field("openai", const=True)
+    model_name: OpenAIModel = Field(
+        ...,
+        description="OpenAI model id (enum value).",
+    )
+
+    def get_model(self) -> BaseLanguageModel:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)